check50.py

```diff
@@ -42,8 +43,13 @@ def main():
 
     # check for newer version on PyPi
     pypi = pypijson.get("check50")
-    if pypi and not args.force and StrictVersion(pypi["info"]["version"]) > VERSION:
-        raise RuntimeError("You are running an old version of check50. Run pip install check50 --upgrade, and then run check50 again!")
+    if pypi and not args.no_autoupdate and StrictVersion(pypi["info"]["version"]) > VERSION:
+
+        # updade check50
+        pip = "pip3" if sys.version_info >= (3, 0) else "pip"
+        subprocess.run([pip, "install", "--upgrade", "check50"])
```

Comment:
If this fails, shouldn't `execv`, should probably just proceed without autoupdating and print warning to `stderr`?